src/app.js

```diff
@@ -1,62 +1,218 @@
-(function() {
-  'use strict';
-
-  angular.module('app', [
-    'ui.router',
-    'ngResource',
-    'hljs',
-    'angularMoment',
-    'foundation',
-    'foundation.dynamicRouting',
-    'foundation.dynamicRouting.animations',
-    'app.projects',
-    'app.builds',
-    'app.jobs',
-    'app.logs'
-  ])
-    .config(config)
-    .run(run)
+/* eslint angular/controller-as: 0, no-unused-vars: 0, angular/log: 0 */
+import angular from 'angular';
+import ngResource from 'angular-resource';
+import 'angular-gantt';
+import hljs from 'angular-highlightjs';
+import angularMoment from 'angular-moment';
+import uiRouter from '@uirouter/angularjs';
+import fastclick from 'fastclick';
+
+import './assets/scss/app.scss';
+
+/*
+ * This file contains the settings for the application.
+ * Settings MUST be stored in a separate directory, because this directory may
+ * be mounted from a different filesystem
+ */
+const baseURL = 'https://cors-anywhere.herokuapp.com/http://acid-api.technosophos.me:7745';
+
+angular.module('app.modules', [uiRouter])
+  .config(routes)
+  .controller('ProjectController', ProjectController)
+  .controller('ProjectBuildsController', ProjectBuildsController)
+  .controller('BuildController', BuildController)
+  .controller('BuildsController', BuildsController)
+  .controller('JobsController', JobsController)
+  .controller('LogController', LogController)
+;
```

Comment:
We'll be able to move these out into their own Angular components/modules later; included in here initially.